README.md

```diff
@@ -83,9 +85,21 @@ The kube-prometheus stack includes a resource metrics API server, so the metrics
 $ minikube addons disable metrics-server
 ```
 
+## Compatibility
+
+### Kubernetes compatibility matrix
+
+|                | Kubernetes < 1.14 | Kubernetes 1.14 | Kubernetes 1.15 | Kubernetes 1.16 | Kubernetes 1.17 |
```

Comment:
I think `Kubernetes < 1.14` might be too strong of a statement as we certainly don't support _all_ versions below 1.14. Was there not a specific kube version we were targeting with `release-0.1`?

Reply:
From https://github.com/coreos/kube-prometheus/releases/tag/v0.1.0
>Note: This will be the last release supporting Kubernetes 1.13 and before.

I believe we can restrict this to Kubernetes 1.13 and drop the support for older versions. What do you think?

Reply:
I think no one uses 1.13 anymore, so lets just go from 1.14 which sadly some vendors still use.

Reply:
In that case, should we keep the `release-0.1` row for deprecation purposes?